public_html/wp-content/themes/pattern-directory/css/components/_pattern.scss

```diff
@@ -25,10 +29,77 @@ body.single-wporg-pattern {
 	.pattern-actions {
 		display: flex;
 		align-items: stretch;
+		flex-wrap: wrap;
+		overflow: hidden;
 
 		button + button {
 			margin-left: 2em;
 		}
+
+		&__notice {
+			margin: 1.5rem 0 0;
+			height: auto;
+			flex-basis: 100%;
+			transform: translateY(0);
+			transition: transform 0.15s ease-in-out;
+
+			// @wordpress/components/notice wraps content
+			> * {
+				display: flex;
+				flex-direction: column; // stack on mobile
+				justify-content: space-between;
+				align-items: flex-start;
+				margin: 0;
+
+				button {
+					margin-top: 0.75rem;
+				}
+
+				@media only screen and (min-width: $breakpoint-small) {
+					align-items: center;
+					flex-direction: row;
+
+					button {
+						margin-top: 0;
+					}
+				}
+			}
+
+			&--is-hidden {
+				margin: 0;
+				flex-basis: auto;
+				height: 0;
+				transform: translateY(60px);
+			}
+		}
+
+		&__guide {
+			max-width: $breakpoint-mobile / 1.25;
+
+			// @wordpress/components/guide overrides
+			height: auto;
+			max-height: none;
+
+			&-content {
```

Comment:
I know these `&{string}` shortcuts are convenient but they make these selectors hard to read, especially here - you need to backtrack 2 levels to figure out the actual class name. Can we keep `&` out of selector strings? Nesting would still be okay, like:

```scss
.foo {
  .bar & { // OK
    color: blue;
  }
  &-bar { // Hard to read
    color: red;
  }
}
```

Reply:
That's fair. Done.